node/src/api/server.rs

```diff
@@ -57,7 +57,8 @@ pub async fn actix_main(config: NodeConfig) -> anyhow::Result<()> {
                     .finish()
             })
             .wrap(Logger::new(LOGGER_FORMAT).exclude("/status"))
-            .wrap(Authentication::new())
+            // TODO: Should we not be using a JWT but rather something more custom?
+            //.wrap(Authentication::new())
```

Comment:
I've disabled the authentication for now as we likely will want to change how we handle this. I imagine we want to use something more custom than the jwt algorithms for the signed token.

Reply:
Yeah, makes sense